### PR TITLE
s3tr: Add README

### PR DESCRIPTION
**Pull request by irq0**

### CODE REVIEW

File: tools/s3tests/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Integrated analysis tools:
+
+- Compare test results to a list of know failing tests for CI purposes
+- Open results in a local [Datasette](https://datasette.io/) instance
+
+## How does test execution work?
+
+- 🧠 s3tr runner inside the s3tr container orchestrates:
+  - for each s3 test divided onto N parallel worker process:
+    - start a 🌉 S3GW container
```
Comment (jecluis) [inline tools/s3tests/README.md]: is this a bridge emoji?

File: tools/s3tests/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+       ghcr.io/aquarist-labs/s3tr:latest \
+       run \
+       --image ghcr.io/irq0/s3tr:latest \
+       --tests s3tests_boto3/functional/test_s3.py::test_copy_to_itself \
+       /out/s3tr.json
+```
+
+`s3tests_boto3` is the path we pull the s3-tests in during image creation
+
+### Run local built without creating a container
```
Comment (jecluis) [inline tools/s3tests/README.md]: s/built/build/ ?

**Review by jecluis — DISMISSED** [review]
generally looks good to me, but I'm very confused by the bridge emoji

File: tools/s3tests/README.md (hunk 1 above)
Comment (irq0) [reply tools/s3tests/README.md]: Yeah. It auto completed from 'gate'. Removing because confusing :)

**Pull request merged**